### ISSUE TITLE
Possibility to freeze the header 

### ISSUE THREAD

**Issue by jburnich** [man-group/dtale#759]
Hi! I just discovered dtale and I am very satisfied with the features it offers. However, I find the fact that the header does not remain fixed when using it quite confusing. Is there an option that would allow to freeze the menu in any circumstance?

<img width="301" alt="Capture" src="https://github.com/man-group/dtale/assets/99728876/897d40f8-4c95-4e4e-951e-e3264bbfa197">

**Comment by aschonfeld** [comment]
Not at the moment, but I can certainly work on adding an option for that

**Comment by aschonfeld** [comment]
@jburnich  so I just published v2.16.0 which includes this functionality.  You can lock the header menu by specifying `lock_header_menu` when calling `dtale.show` (would only affect this piece of data):

```
import numpy as np
import pandas as pd
import dtale

dtale.show(pd.DataFrame([1,2,3,4,5]), lock_header_menu=True)
```

Another would be using global configuration. Following this [documentation](https://github.com/man-group/dtale/blob/master/docs/CONFIGURATION.md) you can add the following to the [app] section for it to be applied to every dataframe you load into D-Tale without specifying parameters in your code.  This would be applied to every piece of data you load into D-Tale:

```
lock_header_menu = True
```

And lastly you can lock the header menu for all data loaded into D-Tale thru code using the following:
```
import numpy as np
import pandas as pd
import dtale
import dtale.global_state as global_state

global_state.set_app_settings(dict(lock_header_menu=True))

dtale.show(pd.DataFrame([1,2,3,4,5]))
```

Also, if you haven't already, please put your ⭐ on the repo when you get a sec. Thanks! 🙏

**Comment by jburnich** [comment]
Good news, thanks for this addition
